package.json

```diff
@@ -72,7 +75,7 @@
     "turbo-test-runner": "0.3.3"
   },
   "engines": {
-    "node": ">=0.10"
+    "node": ">=4.4"
```

Comment:
In the ReadMe[1] and consequently npm page is this information, so make sense change one of the both sides. Or we change the doc or in the code. What/Where do you think it should be better? Is really good we still supporting deprecated and so old versions of the node for this project? 

[1] - https://github.com/feedhenry/fh-fhc#node-version-